armor.js

```diff
@@ -0,0 +1,62 @@
+// levels lookup object
+var levels = { 0: null, 1: 1, 2: 2, 3: 3, 4: 4, 5: 5, 6: 6, 7: 7, 8: 8, 9: 9, 10: 10, 11: 11, 12: 12, 13: 13, 14: 14, 15: 15, 16: "PRI", 17: "DUO", 18: "TRI", 19: "TET", 20: "PEN" }
+
+const success = (item) => {
+  let modObject = item;
+  let enhancementValues = Object.values(levels);
+  let newEnhancement = enhancementValues[modObject.enhancement + 1]
+  modObject = {
+    name: `${modObject.name} +${newEnhancement}`,
+    type: modObject.type,
+    durability: modObject.durability,
+    enhancement: newEnhancement
+  }
+  return modObject
+  // enhancement of zero is not displayed
```

Comment:
Good job grabbing the relevant details from the repo to reference throughout your codebase 👍